exec/java-exec/src/test/java/org/apache/drill/exec/store/bson/TestBsonRecordReader.java

```diff
@@ -88,6 +88,17 @@ public void testTimeStampType() throws IOException {
     assertEquals(1000000l, mapReader.reader("ts").readLocalDateTime().atZone(ZoneOffset.systemDefault()).toInstant().toEpochMilli());
   }
 
+  @Test
+  public void testTimeStampType_now() throws IOException {
```

Comment:
Looks like this is almost the same test as the test before. Can they both be combined, for example, we can write two records (or more) into `BsonDocument`. Also, please use a literal instead of `System.currentTimeMillis()`.

Reply:
fixed